egs/swbd/s5c/local/segmentation/copy_targets_dir.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Copyright    2017  Nagendra Kumar Goel
```

Comment:
Can you move this file to steps/segmentation? I think it will be useful in other cases.

Reply:
Since there is no change to the contents of the targets file, it might be better to copy the scp file and add the suffix to the utterance-ids.

Reply:
@vimalmanohar  I had some other functionality in mind - like creating multiple reps of data for noise, which I apparently didn't finish although the script should work in its current form. How about I move it after I finish that?

Reply:
Even then it would be better to copy the scp file and change the utterance-id. Also I think most scripts expect targets.scp, which is not being created by this script.